### ISSUE TITLE
MYIS Code Migration

### ISSUE THREAD

**Issue by Chris-Dixon-Abt-Global** [cwbi-apps/access-request#23]
## Module Name
MYIS

## Platform 1 or GitHub repository link(s) - note if archive only
https://code.il2.dso.mil/platform-one/products/cwbi/pid/myis

## Module Admin(s): name - username
Kat Jarl - Katherine-jarl
Chris Dixon - Chris-Dixon-Abt-Global 
Justin Eiler - Justin-Eiler-Abt-Global
Andreas Maier - andreasmaier-abt
Emily Savelli - emsavelli

## Contributors (non-admin): name - username
None

**Comment by willbreitkreutz** [comment]
Hi @Chris-Dixon-Abt-Global, I've created the MYIS repo, I think i noted on another ticket, but waiting on profile updates for @Justin-Eiler-Abt-Global.  Otherwise good to go.

**Comment by Chris-Dixon-Abt-Global** [comment]
Thanks Will. I let the three staff we have know that they need to update their profiles per your policy requirements.

[A red and white logo  Description automatically generated]
Chris Dixon, AICP, PMP  |  Associate  |  He/Him/His
+1 303 381 8267

Abt Global  |  One Tabor Center, 1200 17th Street, 10th Floor, Denver, CO 80202
abtglobal.com<https://www.abtglobal.com/>  |  @abtglobalimpact
Abt Associates is now Abt Global

**Comment by Chris-Dixon-Abt-Global** [comment]
Hi @willbreitkreutz , I've added Kat Jarl's username to this request so that she can be added to the repo now: Katherine-jarl 

Thank you!

**Comment by Chris-Dixon-Abt-Global** [comment]
@willbreitkreutz , do you have the link to the repo that you created?

**Comment by willbreitkreutz** [comment]
@Chris-Dixon-Abt-Global yeah, it's https://github.com/cwbi-apps/myis

once you accept the invite to the cwbi-apps org, you should be able to search/see all the repos in the org too.

Katherine should be getting an invite as well.

**Comment by Justin-Eiler-Abt-Global** [comment]
@willbreitkreutz My profile has been updated.

I am unable to access https://github.com/cwbi-apps/myis and have not seen any invites.

Thanks!

**Comment by willbreitkreutz** [comment]
@Justin-Eiler-Abt-Global  you should be getting an invite now. thanks
